### PR TITLE
Features/Improvements/Bugfixes

### PR DESCRIPTION
**Pull request by vikramraman**
- Added OpenTSDB support (MONIT-7390)
- Export GC/Memory stats into Wavefront
- Addressed OOM issues (MONIT-7505): We drop points beyond configured limits for now, will queue to disk and re-send once disk buffering support is implemented.
- Log proxy summary once a minute
- Other minor code improvements

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,9 @@ deps:
 proxy:
 	go build -i -o $(PROXY) -ldflags "$(LDFLAGS)" ./proxy/proxy.go
 
+amd64:
```
Comment (zhizhu7) [inline Makefile]: Besides building for the default OS, can we also include building binary for other OS, like Windows, Linux, darwin?

Comment (vikramraman) [reply Makefile]: Yes, I'm currently working on a ticket to improve the packaging for the proxy. This will be addressed in that.

**Comment by vikramraman** [comment]
- Added a new package and package-all Makefile targets that support building .deb, .rpm linux packages and windows/darwin executables. (MONIT-7576)
- Added config file support for the go-proxy (which is used by the linux systemd wavefront-proxy service) (MONIT-7396)
- Added a --version flag that prints the current version and git tag/commit info for the proxy.

**Review by zhizhu7 — APPROVED** [review]
LGTM.

**Comment by vikramraman** [comment]
Merging to move along the QA process since Zhi has done reviewing it.

**Pull request merged**